common-moonmage.lic

```diff
@@ -49,12 +49,16 @@ module DRCMM
              'You see nothing regarding the future',\
              "You believe you've learned all that you can about",\
              get_data('constellations').observe_finished_messages,\
-             'open it',\
+             'open it','Your vision is too fuzzy',\
              'Roundtime')
   end
 
   def center_telescope(target)
-    DRC.bput("center telescope on #{target}", 'Center what', 'You put your eye', 'open it to make any use of it', 'The pain is too much', "That's a bit tough to do when you can't see the sky", 'Your search for')
+    DRC.bput("center telescope on #{target}", 'Center what',\
+             'You put your eye', 'open it to make any use of it',\
+             'The pain is too much',\
+             "That's a bit tough to do when you can't see the sky",\
+             'Your search for', 'Your vision is too fuzzy')
```

Comment:
❓ Are the slashes `\` necessary at the end of these lines? I'm not a ruby expert and I thought the slahes were only necessary for creating a single string but splitting up the concatenations across multiple lines. I didn't think it'd be necessary if you're formatting the arguments in a method call. 

Ok, did some more googling before posting my question. Per Ruby style guide, I think the `\` are only for [string concatenation](https://rubystyle.guide/#no-trailing-backslash). Method arguments [can be on multiple lines](https://rubystyle.guide/#no-double-indent).

Reply:
I just followed what was already there for def peer_telescope. I could remove the slash and see if it still works.